components/openindiana/nwam-manager/Makefile

```diff
@@ -85,3 +87,14 @@ build: $(BUILD_32)
 install: $(INSTALL_32)
 
 download:: $(SOURCE_DIR)/.downloaded
+
+REQUIRED_PACKAGES += gnome/config/gconf
+REQUIRED_PACKAGES += library/desktop/atk
+REQUIRED_PACKAGES += library/desktop/gtk2
+REQUIRED_PACKAGES += library/desktop/libglade
+REQUIRED_PACKAGES += library/glib2
+REQUIRED_PACKAGES += library/gnome/gnome-libs
```

Comment:
This is a meta-package, consisting of library/desktop/libbonoboui, library/desktop/libexif-gtk , library/desktop/libgnome, library/desktop/libgnomeui, library/desktop/libgtkhtml, library/desktop/startup-notification, text/rarian. What exact packages does nwam-manager require?